src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+server.port=8080
```

Comment:
I recommend to use application.yml instead of properties it is more flexible and cleaner.